src/cqerl_client.erl

```diff
@@ -534,8 +546,23 @@ handle_info({'DOWN', _MonitorRef, process, Pid, _Info}, live, State=#client_stat
             end
     end;
 
+handle_info(heartbeat_check, StateName, State = #client_state{last_socket_send_secs = LastSocketSendTs}) ->
+    {Mega, Secs, _} = os:timestamp(),
+    CurrentTs = Mega * 1000000 + Secs,
+    State1 = case LastSocketSendTs < (CurrentTs - ?HEARTBEAT_MAX_IDLE_TIME_SECS) of
```

Comment:
Can you wrap `LastSocketSendTs < (CurrentTs - ?HEARTBEAT_MAX_IDLE_TIME_SECS)` in some function with a meaningful name?